diff --git a/node_modules/@remix-run/react/components.js b/node_modules/@remix-run/react/components.js
index e29d2eb..b9e57c7 100644
--- a/node_modules/@remix-run/react/components.js
+++ b/node_modules/@remix-run/react/components.js
@@ -650,7 +650,7 @@ let FormImpl = /*#__PURE__*/React__namespace.forwardRef(({
     if (!form) return;
 
     function handleClick(event) {
-      if (!(event.target instanceof HTMLElement)) return;
+      if (!(event.target instanceof Element)) return;
       let submitButton = event.target.closest("button,input[type=submit]");
 
       if (submitButton && submitButton.type === "submit") {
diff --git a/node_modules/@remix-run/react/esm/components.js b/node_modules/@remix-run/react/esm/components.js
index eccf492..3663016 100644
--- a/node_modules/@remix-run/react/esm/components.js
+++ b/node_modules/@remix-run/react/esm/components.js
@@ -626,7 +626,7 @@ let FormImpl = /*#__PURE__*/React.forwardRef(({
     if (!form) return;
 
     function handleClick(event) {
-      if (!(event.target instanceof HTMLElement)) return;
+      if (!(event.target instanceof Element)) return;
       let submitButton = event.target.closest("button,input[type=submit]");
 
       if (submitButton && submitButton.type === "submit") {
